iaso/api/entity.py

```diff
@@ -56,17 +56,24 @@ class Meta:
             "entity_type_name",
             "instances",
             "submitter",
+            "org_unit",
         ]
 
     entity_type_name = serializers.SerializerMethodField()
     attributes = serializers.SerializerMethodField()
     submitter = serializers.SerializerMethodField()
+    org_unit = serializers.SerializerMethodField()
 
     def get_attributes(self, entity: Entity):
         if entity.attributes:
             return entity.attributes.as_full_model()
         return None
 
+    def get_org_unit(self, entity: Entity):
+        if entity.attributes.org_unit:
```

Comment:
entity.attributes might be `None` since the model allows it. Not sure if that happens in the real world (I created such an entity using the Django admin), but something to check IMHO (and if the idea is to always have attributes, let's the model/database enforce it).

Reply:
> Tested on my machine and works fine, well done!
> 
> From a user experience perspective, I wonder if we shouldn't change the condition to show specific columns, so it depend of the entities returned and not of the selected type in the form (=if a search return all entities of the same type => we know which columns to display, regardless of what's selected in "form type").
> 
> I'm thinking for example of the case of users having all entities of the same type: they wouldn't use the form to filter on entity type, but would still expect to see the "correct" columns

Yes but not now, we will have a second round on this by providing the option to change those visible colmumns with the UI

Reply:
> entity.attributes might be `None` since the model allows it. Not sure if that happens in the real world (I created such an entity using the Django admin), but something to check IMHO (and if the idea is to always have attributes, let's the model/database enforce it).

i'll fix that one now